[Ready?] Cult Overhaul. Yep.

tkdrg opened the pull request:
_YES_

Cult is awful, so let's try to improve it. Idea by KorPhaeron. I also
took some inspiration from adrix89's sacrifice cult, so credits to him.
Also thanks a lot specially to MrPerson and Iamgoofball for helping
me brainstorm this, and Joan for the sprites for the summoning orb,
the large nar-sie shell, and the new cult antag hud.

Basically, we remove conversion, and turn cult into a magic version of
nuke, with a small team of stealth elite cultists that have to build a
base in the station and sacrifice people in order to build an army of
constructs and get materials to eventually summon Nar-Sie.

**ANY FEEDBACK IS APPRECIATED**

Tasklist:
- [x] Prototype: Done and tested.
  - [x] Overhaul sacrifice runes (soul stones, summoning orbs~~, rerolling known runes~~)
  - [x] Rebalance tomes, talismans and runes for the 5 starting cult (preliminary)
  - [x] Remove stun talisman REMOVE STUN TALISMAN 
  - [x] Narnar should be the only objective, something like the cult nuke/dom/delta
  - [x] Code the summoning orb: Can either buff the cultist/cult base somehow or be used towards Nar-sie. These orbs should expire in order to force people to use them quickly after the sacrifice.
  - [x] Rebalance Nar-sie: Build a giant demon construct shell in your base. Once it is done, it will announce it for the station. Defend it for long enough in order to summon the Geometer.
  - [x] RNG runes. Each cultist can only write a subset of runes, and a subset of modifiers. Inspiration is basically The Binding of Isaac here, you get a few random powers/items for every sacrifice and you have to make do with them.
- [x] First Playtest Feedback:
  - [x] The shell needs to be able to be summoned more than once, if destroyed, perhaps with some steep cost.
  - [x] There should be more cultists for each pop (higher scaling).
  - [x] Cult, specially early on, is very weak and needs some major rebalancing.
  - [x] Cult supply talismans need more uses.
  - [x] Weak runes should be merged (veil/unveil, deaf/blind/mute).
  - [x] Many more runes should be added. Ideas welcome (added time stop and flame. considering adding a blink rune as well).
  - [x] The large shell doesn't clean up to red alert and such properly on Destroy().
  - [x] Muting/trapping people should be easier, specially for early cult. Stealth is key. (stun landmines added, and curse runes that can mute)
  - [x] The fluff in the "Read Tome" section needs to be improved/updated.
- [x] Second playtest feedback
  - [x] Random runes are too unforgiving? (I don't want to mess with this until a few more playtests to be honest. For now I'll just slightly increase the amount of runes each cultist gets).
  - [x] The shell is too weak to ex_act or singularity_act ~~(bomb immune walls?)~~EVEN BETTER.
  - [x] Defense time should be lowered (I'll try 3 minutes).
  - [x] Orb requirement is too high.
  - [x] More Runes need Talismans.
- [x] Suggestions/Requests:
  - [x] Cult traps. These tie into the rune overhaul, and help prevent meta from assistants prowling maint roundstart looking for bases. These should be obtainable from the supply talisman. (kind of done, with stun traps)
  - [x] Maybe summoning orbs probably don't need a timeout?
  - [x] The defense timer could perhaps do with being slightly shorter (3 minutes?).
  - [x] Cult could use some stealthy ritual dagger with bleed effects and stuff?
  - [x] ~~Maybe do some bullshit like "the planets align" so they have to sac in X time window so we dont run into the gang problem of "kill everyone and then dom" (perhaps using the Sun controller).~~ Not needed right now.
  - [x] ~~Rune overhaul: modifiers that change how neighboring runes work. Could reuse the cult structures and runed metal, or some other material with various tiers. Example modifiers: Makes it hit area of effect, but weaker effects the further out it goes. Doubles area but causes you to take a good chunk of damage on activation. The point is for people to experiment with runes to get their favorite effect.~~ Too much effort for too little gain. Of course, anyone else is more than welcome to do it once I push this.

tkdrg committed:
Cult Overhaul

Cult is awful, so let's try to improve it. Idea by KorPhaeron. I also
took some inspiration from adrix89's sacrifice cult, so credits to him.
Also thanks a lot specially to MrPerson and Iamgoofball for helping
me brainstorm this, and Joan for the sprites for the summoning orb,
the large nar-sie shell, and the new cult antag hud.

Basically, we remove conversion, and turn cult into a magic version of
nuke, with a small team of stealth elite cultists that have to build a
base in the station and sacrifice people in order to build an army of
constructs and get materials to eventually summon Nar-Sie.

Stun talismans and conversion are gone. Nar-Sie is now the only
objective. Sacrifice runes now provide summoning orbs, which you can use
to drop a large shell. Insert enough orbs in the shell and you trigger a
Delta. Defend the shell for 3 minutes and Nar-sie will arise. This shell
is bombproof and singularityproof. It may be summoned up to three times
if destroyed, but you will have to start from the beginning sac-wise.

Runes are now RNG. Each cultist only gets 50% of the runes. Inspiration
is basically The Binding of Isaac here (thank you MrPerson for this!).
Furthermore, many old runes were merged/removed and other new ones were
added. Almost all were massively rebalanced. Cult now also has a
stealthy ritual dagger with high bleeding and throw effects.

RemieRichards commented:
For the rune modifiers, they should be runes that aren't circular, and when drawn next to a rune add to the design of that rune, and dont function on their own.

AlexanderUlanH commented:
Suddenly, resurrection runes are worth using.

Anonus commented:
Possible rune modifiers;
Making a rune trigger when crossed by noncultists
Making nearby runes trigger when a trigger rune is crossed
Making a rune constantly active but very weak effects

Also, modifiers should probably take up a 'slot' so you can't just slap mods in until you've got a super rune, but I'm guessing that's already planned and just not stated.

Supermichael777 commented:
^ blood boil landmines.
all the ![YES!](https://media.giphy.com/media/FjeGBljESVAzu/giphy-facebook_s.jpg)

Bawhoppen commented:
I liked the concept of religious fanaticism and people forcefully indoctrinating into the cult, like an actual cult but with actual cult magic unlike irl

duncathan commented:
A couple of my friends are in a cult and they don't try to forcefully indoctrinate me

No really, I'm serious

Bawhoppen commented:
cult is a spectrum
clearly this cult is worshipping evil dark narsie so they will try and force convert you and murder you and create demon constructs

duncathan commented:
Who are you to call the Geometer evil?

RemieRichards commented:
She just wants to be loved.
her love involves sacrificing humans and pouring blood everywhere.
what's wrong with that?

Zilenan91 commented:
![m bison](https://cloud.githubusercontent.com/assets/13275673/11733092/14db3220-9f5f-11e5-8b6a-c7c2def66989.png)

10/10 IGN MERGE IMMEDIATELY

Robustin commented:
This literally just sounds like you're turning Cult into Hand of God.

Anonus commented:
Hand of god has conversion and awful communication, as well as much less obvious objectives. It's more like gang, with magic instead of guns, only one gang, and no conversion.

Anonus commented on code/game/gamemodes/cult/cult_structures.dm:
```diff
@@ -1,3 +1,142 @@
+	"<span class='danger'>You punch [src].</span>")
+	damaged(5)
+
+/obj/structure/constructshell/large/attackby(obj/item/weapon/I, mob/living/user, params)
```
You have two attackby procs, so you can't add summoning orbs to the shell.

Anonus commented on code/game/gamemodes/cult/cult_structures.dm:
```diff
@@ -1,3 +1,142 @@
+	var/orbs_needed = 1
+	var/animate_alpha = 0
+	var/image/black_overlay = null
+	var/time_to_win = 120 //2 minutes
```
This appears to actually be about a second.

duncathan replied:
how'd that happen

tkdrg replied:
it's 12 seconds since it's measured in ds. i didn't actually test any of this yet, so

Anonus replied:
It's actually less than 12 seconds, which doesn't make any sense(it looks like it should be 12) but it's like 1.2 seconds.

Anonus commented on code/game/gamemodes/cult/cult_structures.dm:
```diff
@@ -1,3 +1,142 @@
+		qdel(O)
+		update_icon()
+	if(orbs >= orbs_needed)
+		start_takeover()
```
This should probably be indented under the orb check.

Robustin commented:
TIL cult doesn't have conversion

tkdrg commented:
This overhaul proposes removing conversion. My justification is that stun papers are cancer, however it's very hard to convert without them. Thus, this removes both.

Zilenan91 commented:
@tkdrg So will the cultists start as part of the crew or will they spawn in their own culty spaceship to then warp to the station from

tkdrg commented:
They spawn as part of the crew. It is expected that they ought to build some secluded base on the station and defend it in order to summon narnar.

Zilenan91 commented:
So will you be making it much easier for them to get cult armor/swords? Because they're really gonna need it.

tkdrg commented:
It's still the same, except now only a third of the cult can scribe the rune. I don't see how "write rune, click" is difficult.

Zilenan91 commented:
So only 2 guys can actually make armor runes?

Edit : Ah okay I see that last bullet point. So how this would work is the cult sacrifices a dude and then gets additional rune modifiers for the one who sacced?

tkdrg commented:
That's the idea eventually, although that isn't implemented right now (it's the third to last bullet point). Right now, you have to ask the other cultists for help, or rely on your supply talisman.

Zilenan91 commented:
That sounds like a good system. Is there going to be anything in place to prevent a cult from all getting the same rune modifiers when they sac people?

tkdrg commented:
The plan is making it all random, so I guess so.

Anonus commented:
My gut feeling is that 2 minutes is definitely too short, and it should be like 4-5. The timer should also display in the stat panel so it's not totally unclear when it's going to go nar-sie shaped.

tkdrg commented:
2 minutes was just a guess. I'll make it 4. I didn't display anything because It's a spooky scary cult that doesn't care about petty human measurements such as the concept of "time" (and also because nuke doesn't have one either and it just makes everything more despair).

Anonus commented:
In nuke, though, there's no real chance you're going to be able to defuse the nuke, but here there's a good chance you can destroy the shell.

bgobandit commented:
The "read tome" section needs to be updated

Zilenan91 commented:
Yeah like I said before if the cultists are weak early game then we can make them spawn on a culty ship on the centcom zlevel full of some cult armor and other gear to start them off that they can tp to the station from. They shouldn't be able to go back to the ship though because I really don't want to stay in the days where cultists leaving the zlevel is an acceptable thing to happen.

Bawhoppen commented:
I'd like to restate how much I think this is cancer and while it might be better than our current cult, the core concept is way worse.

Zilenan91 commented:
@Bawhoppen Baw, give actual criticism. All you do is say things are shit but never say why, this kills the development.

Bawhoppen commented:
I already gave my reasons, there was a lack of opposition in this PR so I had to restate my own.

Zilenan91 commented:
so you're opposing it for oppositions sake

Bawhoppen commented:
You didn't understand what I meant. I gave my reasons earlier in this PR, and those points weren't really addressed, so I was bringing it up again.

Zilenan91 commented:
You didn't even give actual reason for it, you just said that you liked the current concept of cult better, then proceeded to say this is the worst thing ever

Bawhoppen commented:
Just because you don't agree with the reason doesn't invalidate it.

tkdrg commented:
Well, I didn't specifically address it because it's basically in the opening, but my reasoning for removing conversion is simple:
* We have plenty of gamemodes that fill the "easy and quick conversion" niche.
* It is very difficult to maintain the "difficult conversion" part of cult without stunpapers.
* Stunpapers promote unpleasant gameplay, and should be removed.
Thus, we've come to the conclusion that we could attempt to remove both conversion and stunpapers. It's compatible fluff-wise, and given that we have constructs, it doesn't remove people from the round. This is why I think it's a solid plan.

Bawhoppen commented:
I can sort of understand that perspective, but I still personally think that cult should be the conversion gamemode above others, and it should be more integrated into the station's population itself, rather than an outside force. That also leads to religious inquisitions and fun like that.

tkdrg commented:
Well, it is integrated in the station's population. The basic idea was "on-station nuke". I don't mind conversion, but I can't think of a solid way to keep it, and I believe that, given the amount of conversion gamemodes we have, we won't lose much by switching one over.

Bawhoppen commented:
Its mainly the execution of current cult thats the issue, and while I couldn't be certain to know how to fix it, I'd disagree with changing the concept.

tkdrg commented:
I don't know, I don't think the concept changed that much. The focus just switched over from conversion to sacrifice. I think the issue is that its conversion forced people to use stuns, and stun-based combat is dull. So this provides players with a different kind of "conversion", one that lets you deal with your target in any way you wish before "converting" them.

Zilenan91 commented:
So can we make a very costly rune that allows us to revive dead constructs?

tkdrg commented:
Well I mean, right now reviving a cultist costs a dead body, but for constructs if we made it so it would probably be better to just make a new one.

Zilenan91 commented:
Yeah just recovering the soul shard then, but not an easy process

KorPhaeron commented:
Maybe you should give cultists free comms/a hivemind. A stealthy team needs coordination, and having to hold a "please kill me" book in your hand to talk doesn't facilitate that.

Also randomly make one of the cultists a leader and give him BIG TEXT

tkdrg commented:
Innate comms is fair, I guess. Not sure about a leader, though. I guess we can test with comms only, and if it isn't enough I'll add a leader.

optimumtact commented:
hahaah why don't we just make nukeops happen as twice as much and just remove cult?

because this is just themed nuke ops

Zilenan91 commented:
@optimumtact Gonna assume you're being serious here, but it's literally nothing like Ops outside of no conversion

optimumtact commented:
>Basically, we remove conversion, and turn cult into a magic version of
nuke, with a small team of stealth elite cultists that have to build a
base in the station and sacrifice people in order to build an army of
constructs and get materials to eventually summon Nar-Sie.

okay.jpg

Zilenan91 commented:
If you removed any mention of nukes from that sentence does it sound anything like Ops

CosmicScientist commented:
@Zilenan91. Small team (nuke ops), stealth (don't deny it, nuke ops), elite (nuke ops).

Zilenan91 commented:
Stealth not so much since they'll go loud quickly and have the tools to sustain themselves as well as having very powerful runes, small team I guess shadowlings are ops now, and elite, well op borgs sometimes talk over binary immediately after activation so that's not really a thing

CosmicScientist commented:
If you're going to say nuke ops aren't elite because of player actions then no role is elite in this game.

Anonus commented on code/game/gamemodes/cult/cult_structures.dm:
```diff
@@ -1,3 +1,135 @@
+	if(istype(O, /obj/item/summoning_orb) && (orbs < orbs_needed))
+		visible_message("<span class='cult'>\The [src] glows.</span>")
+		orbs++
+		qdel(O)
```
Shouldn't this be the last thing done in this proc?

tkdrg replied:
Not really because these don't have force. I'll just add a return after it.

Anonus commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -729,52 +542,33 @@ var/list/teleport_other_runes = list()
-	cultist_name = "Deafen"
-	cultist_desc = "Causes all non-followers nearby to lose their hearing."
+	cultist_name = "Curse"
+	cultist_desc = "Causes all non-followers nearby to lose their hearing, sight and voice."
```
Theme; call it Debilitate with the admittedly kind of silly secondary name 'Rite of the Shadowed Mind'

Anonus commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -729,52 +542,33 @@ var/list/teleport_other_runes = list()
 	for(var/mob/living/carbon/C in viewers(src))
 		if(!iscultist(C) && !C.null_rod_check())
-			C << "<span class='cultitalic'><b>You can't see!</b></span>"
+			C << "<span class='cultlarge'>You feel a dark energy violate your brain.</span>"
```
Along with that, 'You feel oily shadows cover your senses.'

Anonus commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -981,6 +771,27 @@ var/list/teleport_other_runes = list()
 
+//Rite of the Cleansing Flame: Sets all non-cultists on fire.
+/obj/effect/rune/flame
+	cultist_name = "Flame"
```
Call it Immolate, secondary name is fine.

Anonus commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -981,6 +771,27 @@ var/list/teleport_other_runes = list()
+/obj/effect/rune/flame
+	cultist_name = "Flame"
+	cultist_desc = "Sets any non-believers who can see the rune on fire."
+	invocation = "Bh'rt an'fi'ah!"
```
"Dedo va'batoh!"

Anonus commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -981,6 +771,27 @@ var/list/teleport_other_runes = list()
+	visible_message("<span class='warning'>[src] burns away, scorching the floor below!</span>")
+	for(var/mob/living/carbon/C in viewers(src))
+		if(!iscultist(C) && !C.null_rod_check())
+			C << "<span class='cultlarge'>You feel an unholy presence entering your veins.</span>"
```
'You feel your skin crisp as you burst into flames!'

Anonus commented on code/game/gamemodes/cult/runes.dm:
```diff
@@ -1027,3 +837,39 @@ var/list/teleport_other_runes = list()
+/obj/effect/rune/timestop/invoke(mob/living/user)
+	visible_message("<span class='warning'>[src] flares up for a moment, and then disappears into itself!</span>")
+	new /obj/effect/timestop/cult(get_turf(src))
+	for(var/mob/living/carbon/M in orange(1,src))
```
So you stand on the rune and don't take damage? USE RANGE

babinsquared commented:
>Rebalance Nar-sie: Build a giant demon construct shell in your base. Once it is done, it will announce it for the station. Defend it for long enough in order to summon the Geometer.

I'm just gonna throw this out there because it caught my attention: I feel like the game is pretty high on the number of "announce the thing, then defend the thing" wincons right now. Like after a certain number of antags use this, it gets old.

tkdrg commented:
Well, the gamemodes that currently do that are Gang and Malf. Meanwhile, we have 4 conversion gamemodes (Rev, Cult, Gang, Shadowling).

CosmicScientist commented:
Swapping out conversion for defence doesn't make it the right direction. There are other things cult can do to summon Nar-Sie, including in a cult's theme rather than in game mechanic theme.

As a thing I just want to say out of confusion, gang and shadowling has conversion to stop plaeyrs from being thrown out of the round by being killed or so that seemed to be all anyone said about that aspect. Are we going to go back to a desire of preventing players from being kicked to ghostchat if this gets through?

Anonus commented:
Five, Hand of God has conversion too. Though shadowling and Hand of God are both disabled right now, iirc.

Zilenan91 commented:
Why's shadowling disabled?

Also yeah cultists can sac people and turn them into constructs, so they aren't totally out if they get dunked on. Unless the construct they got turned into dies, in which case rip.

Amemelius commented:
Using a soul stone on someone should *probably* leave a summoning orb if it doesn't already (I don't think it does).

Zilenan91 commented:
It doesn't

Anonus commented on code/game/gamemodes/cult/ritual.dm:
```diff
@@ -178,11 +124,8 @@ It also contains rune words, which are soon to be removed.
+	text += "<font color='red'><b>Debilitate</b></font><br>The Rite of the Shadowed Mind is simple. When invoked, it will cause all non-cultists that can see its rune to become deaf, blind and mute for a \
+	considerable amount of time.<br><br>"
 
 	text += "<font color='red'><b>Stun</b></font><br>Though the Rite of Blazing Light is weak when invoked normally, using it in conjuction with the Rite of Binding makes it much more useful. \
```
You can't make stunpapers anymore so the "use this with imbue to stun the crew forever" is wrong. You should note that it stuns on cross now instead, something like this;

    Though the Rite of Blazing Light is weak when invoked normally, merely disorienting and disabling nearby non-cultists, non-cultists that step on the rune will be stunned without expending its power.
    However, the disorientation caused by this rune is quite brief, and when invoked directly, it will not remain for further invocation.

Robustin commented:
So cult are supposed to construct their own base on station and defend it without any reliable stuns or conversion mechanic.

That was all I needed to hear to support scrapping this entirely. It's one thing to have a small closet you can use to sac/convert people, but a base suited for 5 cultists where you're going to summon a god and hold off tens of dozens of crew who will try to stop you.

Like I said before either you have to set Cult up with obscenely strong stuff, stuff that would make nukeops and their syndieborgs go "oh shit thats really strong", and in doing that you're going to either make them about as fun as fighting syndieborgs (average crew won't stand a chance, going to require powergayman with lube or TTV's if you're not sec) or you're going to make them all insufficiently strong and cult gets steamrolled and pretty soon its off rotation for being unplayable.

None of this is worth fucking with a solid game mode with its own niche, go screw around with malf if you want to overhaul a mode that is terrible in almost every respect.

Robustin commented:
Plus did I mention like one of the core themes of the 'cult' archetype is their desire to grow their numbers through conversion?

tkdrg commented:
> So cult are supposed to construct their own base on station and defend it without any reliable stuns or conversion mechanic.

They have reliable stuns (stun rune, time stop rune, debilitate rune to some extent). They just no longer have any *portable* stuns. As I've said in the original post, this overhaul was because I (and most people I've talked to) found stunpaper's insta stun+mute really unfun. However, conversion is very difficult without stun papers, so it was suggested to me that it might be viable to just remove it.

> That was all I needed to hear to support scrapping this entirely. It's one thing to have a small closet you can use to sac/convert people, but a base suited for 5 cultists where you're going to summon a god and hold off tens of dozens of crew who will try to stop you.

That's a valid concern and it's part of the reason why I removed the 5-minute time limit on the orb. It means you can decentralize your saccing and then collect all the cultists on a place when you want to summon. The cults don't usually get steamrolled by this point because they generally have a veritable army of constructs, which tends to hold off the crew for long enough (barring BoHs/TTVs, which have been somewhat dealt with).

> Like I said before either you have to set Cult up with obscenely strong stuff, stuff that would make nukeops and their syndieborgs go "oh shit thats really strong", and in doing that you're going to either make them about as fun as fighting syndieborgs (average crew won't stand a chance, going to require powergayman with lube or TTV's if you're not sec) or you're going to make them all insufficiently strong and cult gets steamrolled and pretty soon its off rotation for being unplayable.

The balance is a serious issue, and honestly I'm afraid of it because I'm not great at balance. The playtests right now show that cults are generally really weak, even though I've made some gigantic buffs, just because so many things can go wrong early on and then you end up with red alert shuttle and you have 7 minutes to sac 5 people. However I don't want to buff them too much: a single powergamer cultist can basically murderboner everyone right now already, I'd rather not make that worse. I want to think that people suck due to their lack of experience with the overhaul, but then we have the counter argument of the near-constant fluke ops we keep seeing. The worst part is that I can't think of an easy way to solve this problem. Conversion would reduce it to some extent once you get the converts going, but then you get the issue of "how do we start converting": either we make conversion fast and easy (like rev or gang), or we give the cult a very strong portable stun, which I really don't want to do. Suggestions on how to fix this are welcome, I'm willing to change everything except for the removal of stun paper as that's the whole point of this PR.

> None of this is worth fucking with a solid game mode with its own niche, go screw around with malf if you want to overhaul a mode that is terrible in almost every respect.

Malf is awful too, but I really really hate playing AI and I'd rather not try to improve something that I can't understand. Cult on the other hand I can kind of enjoy apart from the bullshit stunpapers, yet it is generally agreed upon that it is one of our worst gamemodes, so that's why I picked it.

> Plus did I mention like one of the core themes of the 'cult' archetype is their desire to grow their numbers through conversion?

Yes, and it would be nice if we could keep conversion, but gameplay matters more than fluff. If conversion is forcing us to have the toxic gameplay brought upon by the stunpapers then I think removing it is the better choice.

Bawhoppen commented:
:capital_abcd:  Wall of text :capital_abcd: 

>Yes, and it would be nice if we could keep conversion, but gameplay matters more than fluff.

Theres a difference between core concept and fluff, and I think the line was hit in this case.

>a single powergamer cultist can basically murderboner everyone right now already, I'd rather not make that worse. I want to think that people suck due to their lack of experience with the overhaul, but then we have the counter argument of the near-constant fluke ops we keep seeing.

That kind of thing is sort of inherent to ss13, and I don't think it could be fixed unless we did something drastic.

Came up with some kind of a solution.

A final solution.



....


Roleplay.

tkdrg commented:
Added the cult dagger and squashed commits. Should be ready-ish to go I believe, pending another testmerge for playtesting (I already asked Kor) and Robustin's response to my answer.

Anonus commented:
I probably should warn you that keeping the new cult icon is probably unwise just because of how hard to see it can be aND I LOST TO A GREEN SQUID SEND ME TO HELL

tkdrg commented:
but i dont like the green squid i want yours

KorPhaeron commented:
I'll put this up in a bit but I want to test nuke challenge as well

tkdrg commented:
it's up already actually, I asked hg and he put it. But sure feel free to take it down.

Robustin commented:
My thoughts on this haven't changed but I'm getting pretty tired of all these utterly broken modes being shoved into rotation, especially at the expense of actually working modes (like cult).

90% of the stuff you've "learned" so far could've been expressed to you in 10 minutes from an experienced player who has access to a straightforward yet detailed (i.e. all numbers/damage figures included) summary of what this mode entails. Instead we get dozens of clusterfuck rounds where nobody knows whats going on. 

I would much rather that you go work with whoever's handling Hand of God right now and merge it into one functioning "god worshippers who build a magical base and hold off their enemies while making sacrifices to their god" mode rather than two modes that are broken and nowhere near playable status.

This thing will never get a positive review from me while it REPLACES cult, but I'll reserve further remarks for when I test the latest iteration.

Bawhoppen commented:
Its almost like any gamemode we have made ourselves is shit

tkdrg commented:
> My thoughts on this haven't changed but I'm getting pretty tired of all these utterly broken modes being shoved into rotation, especially at the expense of actually working modes (like cult).

I dunno, I really don't think cult is fine as it is. I mean, yeah, it *works*, but most people generally don't enjoy either being cultists or being loyalists in this gamemode (if the amount of prayers asking for deculting and the speediness of the shuttle calls are anything to go by). And further, this isn't utterly broken. It's different, true, but it works, too, and I personally found it more fun.

> 90% of the stuff you've "learned" so far could've been expressed to you in 10 minutes from an experienced player who has access to a straightforward yet detailed (i.e. all numbers/damage figures included) summary of what this mode entails. Instead we get dozens of clusterfuck rounds where nobody knows whats going on.

Perhaps, but that hadn't been suggested to me before. The six playtests on Sybil were rather confusing for the players, true, but by the sixth all cultists played fairly well and they had a pretty normal round, so I'd think that this is more or less just because it's a new thing.

> I would much rather that you go work with whoever's handling Hand of God right now and merge it into one functioning "god worshippers who build a magical base and hold off their enemies while making sacrifices to their god" mode rather than two modes that are broken and nowhere near playable status.

I did work with Remie on HoG for a while, but to be honest I wasn't satisfied with the end result. HoG had more of a focus on fast conversion and RTS-like mechanics, I'd say that it is much closer to blob+rev than to this cult or to the current cult. The "hold off your enemies" mechanic is here because I believe that it is very cathartic for the players to have a final showdown and a chance at changing the destiny of the round, rather than having the big bad round ender be off-screen and then have dozens of people asking "It was cult?"/"It was shadowling?"/"It was ops?" after the round ends. I don't see the issue with reusing an idea, if it is a good one. I think this is still sufficiently different from the other gamemodes with base defense (gang and malf).

> This thing will never get a positive review from me while it REPLACES cult, but I'll reserve further remarks for when I test the latest iteration.

Well, I can't change your opinion about that, I guess. There would be no meaning in having two cults, and I'd rather not discard this one.

Zilenan91 commented:
Looking good, far, far more enjoyable to play against and as.
